src/mon/MonCommands.h

```diff
@@ -348,10 +349,10 @@ COMMAND("mds compat rm_incompat " \
 COMMAND_WITH_FLAG("mds add_data_pool " \
 	"name=pool,type=CephString", \
 	"add data pool <pool>", "mds", "rw", "cli,rest", FLAG(OBSOLETE))
-COMMAND_WITH_FLAG("mds remove_data_pool " \
+COMMAND_WITH_FLAG("mds rm_data_pool " \
```

Comment:
this sequence of removal/addition seems weird. Am I reading this wrong, or are we effectively removing and adding lines that are essentially the same, only at different points of the code?

Reply:
The reason that one looks a little funny is that I tried to make things show up in a consistent order, with "rm" being first before the now-deprecated "remove". The intent, though, is that both commands should be supported, and the "remove" (or "delete" or "del") forms be flagged deprecated.